package.json

```diff
@@ -42,7 +42,7 @@
     "path-to-regexp": "^3.0.0",
     "pretty-error": "^2.0.0",
     "prop-types": "^15.0.0",
-    "qs": "^5.2.0",
+    "qs": "^6.0.0",
```

Comment:
how about installing `qs@^6.5.2` since `@twreporter/redux@5.0.0-beta.11` requires it.